src/OpenTelemetry.Exporter.Jaeger/ApacheThrift/TException.cs

```diff
@@ -32,5 +33,10 @@ public TException(string message, Exception inner)
             : base(message, inner)
         {
         }
+
+        // A constructor is needed for serialization when an
```

Comment:
Agree with the comment, but these Thrift Exceptions aren't being propagated are they?

Reply:
I don't think they are. Added the seriazlization following [guide @reyang linked](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/exceptions/creating-and-throwing-exceptions#defining-exception-classes) in #1043 but I see now that we were mostly referring to that for exception message consistency. I'll remove this unless there are other thoughts

Reply:
lets leave these files untouched